src/Classes/ServiceAPI/MyRadio_Scheduler.php

```diff
@@ -143,6 +150,36 @@ public static function getTerm($termid)
 
         throw new MyRadioException('That term could not be found', 400);
     }
+    
+    /**
+     * Returns the current term, if there is one.
+     *
+     * @todo There's currently no caching on this, could be a potential slowdown
+     *
+     * @return Array of a term
+     */
+    
+    public static function getActiveTerm()
+    {
+        return self::$db->fetchAll(
+            'SELECT termid, EXTRACT(EPOCH FROM start) AS start, descr
+            FROM terms
+            WHERE start <= now() AND finish > now()
+            ORDER BY start ASC'
+        );
+    }
+    
+    /**
+     * Returns if we are currently in term time.
+     *
+     * @todo There's currently no caching on this, could be a potential slowdown
+     *
+     * @return Boolean
+     */
+    public static function isActiveTerm()
```

Comment:
"isActiveTerm" might suggest that you're querying a particular term or something. Maybe `isInActiveTerm` or just `inTerm` would do?

Reply:
I changed it to isTerm() to keep the somewhat convention of "is*" when there's a boolean output.